PWGDQ/Core/VarManager.h

```diff
@@ -865,6 +865,14 @@ void VarManager::FillPropagateMuon(const T& muon, const C& collision, float* val
     values[kEta] = propmuon.getEta();
     values[kTgl] = propmuon.getTgl();
     values[kPhi] = propmuon.getPhi();
+    std::cout << "DCAs before" <<std::endl;
+    std::cout << "DCAx: " << values[kMuonDCAx] <<std::endl;
+    std::cout << "DCAy: " << values[kMuonDCAy] <<std::endl;
+    values[kMuonDCAx] = propmuon.getX() - collision.posX();
+    values[kMuonDCAy] = propmuon.getY() - collision.posY();
+    std::cout << "DCAs after" <<std::endl;
+    std::cout << "DCAx: " << values[kMuonDCAx] <<std::endl;
+    std::cout << "DCAy: " << values[kMuonDCAy] <<std::endl;
```

Comment:
please remove the printouts, this will create very large log files for everyone. For debugging at this level i suggest to just run locally over a few aod files

Reply:
Thanks for your suggestion! I will remove them.